Tests/Harness/AppDelegate.swift

```diff
@@ -3,8 +3,12 @@ import UIKit
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
   var window: UIWindow?
+  #if swift(>=4.2)
+    typealias UIApplicationLaunchOptionsKey = UIApplication.LaunchOptionsKey
```

Comment:
What do you think about expressing this in terms of the new API instead of the old API — i.e., a conditional extension on `UIApplication`?